### PR TITLE
DOCSP-40422: v2.26 release

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-40422>
Staging:
[Whats new](https://preview-mongodbrustagir.gatsbyjs.io/csharp/DOCSP-40422-v2.26-release/whats-new/#what-s-new-in-2.26)
[Compat](https://preview-mongodbrustagir.gatsbyjs.io/csharp/DOCSP-40422-v2.26-release/compatibility/)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -15,10 +15,11 @@ What's New
    :values: reference
 
 .. meta::
-   :keywords: update, new feature, deprecation, upgrade, driver v2.19, driver v2.20, driver v2.21, driver v2.22, driver v2.23, driver v2.24, driver v2.25
+   :keywords: update, new feature, deprecation, upgrade, driver v2.19, driver v2.20, driver v2.21, driver v2.22, driver v2.23, driver v2.24, driver v2.25, driver v2.26
 
 Learn what's new in:
 
+* :ref:`Version 2.26 <csharp-version-2.26>`
```
Comment (shuangela) [inline source/whats-new.txt:22]: Question: Why is this csharp-version-2.26 while the other links are version-2.25, 2.24, 2.23, etc? Not a blocker but curious about consistency.

**Review by shuangela — APPROVED** [review]
LGTM! Small comment.

File: source/whats-new.txt (hunk 1 above)
Comment (rustagir) [reply source/whats-new.txt:22]: In the past, we have not been consistent with including the product name in these anchors, but this is important as we sometimes have cross-repo links and this ensures that there wont be collisions/ambiguous links.

**Review by BorisDog — APPROVED** [review]
LGTM

File: source/whats-new.txt (hunk 1 above)
Comment (shuangela) [reply source/whats-new.txt:22]: Got it, thanks!

**Pull request merged**